Userland/Libraries/LibJS/Runtime/Temporal/Calendar.h

```diff
@@ -72,6 +72,6 @@ i32 iso_year(Object& temporal_object);
 u8 iso_month(Object& temporal_object);
 String iso_month_code(Object& temporal_object);
 u8 iso_day(Object& temporal_object);
-Object* default_merge_fields(GlobalObject&, Object& fields, Object& additional_fields);
+Object* default_merge_fields(GlobalObject&, Object const& fields, Object const& default_merge_fields);
```

Comment:
Looks like there's a bit of confusion about the naming of the second variable in this commit, it's called `additional_fields` in the `cpp` file, `default_merge_fields` here in the header file, `default_merge_fields` in the commit title, and `additional_fields` in the commit body, we should probably choose one and stick with it :)

Reply:
Oops, I think I accidentally copy & pasted the function name into the arguments, and then got confused when writing the commit message. Fixed, thanks for catching this :^)